apps/src/templates/progress/progressTestHelpers.js

```diff
@@ -286,6 +286,5 @@ export const fakeProgressTableReduxInitialState = (
     },
     unitSelection: {scriptId: scriptData.id},
     locales: {localeCode: 'en-US'},
-    isRtl: false,
```

Comment:
Curious about how this comes into play? I don't see where the banner uses it

Reply:
Needed it to pass tests when I first made this change...for some reason 🤔

Reply:
Got it- I see now how Notification.jsx uses it (just not passed as a prop). Makes sense that if it wasn't here before the banner, nothing else would need it now that the banner will be one.